examples/example-react-task/package.json

```diff
@@ -27,10 +27,13 @@
   },
   "dependencies": {
     "@realm/react": "^0.5.1",
+    "bson": "^5.3.0",
```

Comment:
We do provide `bson` in the `realm` package directly. Usually it's used via e.g. `import { BSON } from "realm"`. But it'd also be interesting here to know the error you got from not have this separately.